phpstan.neon

```diff
@@ -1,8 +1,13 @@
 parameters:
+    level: 4
+    paths:
+        - src
+        - tests
     ignoreErrors:
         - '/Constructor of class Sentry\\HttpClient\\Encoding\\Base64EncodingStream has an unused parameter \$(readFilterOptions|writeFilterOptions)/'
         - '/Call to an undefined method Sentry\\ClientBuilder::methodThatDoesNotExists\(\)/'
         - '/Argument of an invalid type object supplied for foreach, only iterables are supported/'
+        - '/Binary operation "\*" between array and 2 results in an error\./'
```

Comment:
This is not fixable because it's an intentional error in the tests to provoke the error handler invocation. See https://github.com/getsentry/sentry-php/blob/ad79d912603c235494dae99cc331fa8f52827ba6/tests/AbstractSerializerTest.php#L402-L404